### PR TITLE
RAW Type Added

### PR DESCRIPTION
**Pull request by Auguronomics**
Allow to work with RAW datatype 

!|Execute|create table raw_line(objectid integer, myGuid RAW)|

!|Insert |raw_line |
|objectid|myGuid |
|10      |0ce894b4-bf19-e511-840f-0050568969e2|

!|Query  |select objectid, RAW from spatial_line|
|objectid|myGuid? |
|10      |0ce894b4-bf19-e511-840f-0050568969e2|

### CODE REVIEW

**Comment by MMatten** [comment]
As `RAW` is a binary type, like `BLOB` would `byte[]` be a more appropriate, than `String` Java type for this? There is also `oracle.sql.RAW' available in the JDBC driver.

**Comment by MMatten** [comment]
We could do with an acceptance test for this added into:

`FitNesseRoot/DbFit/AcceptanceTests/JavaTests/OracleTests/FlowMode/DataTypes`

**Comment by javornikolov** [comment]
I'm not sure why are these travis.yml updates - if nothing else, seems something out of scope of the RAW data type support.

File: dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java [hunk 1]
```diff
@@ -390,7 +390,7 @@ private CallableStatement openDbCallWithParameters(String query,
     // map types
     private static List<String> stringTypes = Arrays.asList(new String[] {
             "VARCHAR", "VARCHAR2", "NVARCHAR2", "CHAR", "NCHAR", "CLOB",
-            "NCLOB", "ROWID", "BOOLEAN"
+            "NCLOB", "ROWID", "BOOLEAN","RAW"
```
Comment (javornikolov) [inline dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java]: Missing space after the `,`.

Comment (Auguronomics) [inline dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java]: I correct the space lack, now

**Comment by javornikolov** [comment]
> As RAW is a binary type, like BLOB would byte[] be a more appropriate, than String Java type for this? There is also oracle.sql.RAW available in the JDBC driver.

Also, since RAW may contain non-printable characters, maybe it makes sense to use e.g. HEX representation on the test pages.

**Comment by Auguronomics** [comment]
RAW looks like CLOB,
I'll add the dbfit Documentation soon, copy paste from the post comment.
I'm in trouble to download the travis build form S3.

**Comment by javornikolov** [comment]
> RAW looks like CLOB,

It's more like BLOB. (We don't have BLOB support yet either).

**Pull request closed without merging**